src/renderers/testing/ReactTestRenderer.js

```diff
@@ -35,92 +46,65 @@ function getRenderedHostOrTextFromComponent(component) {
   return component;
 }
 
+class ReactTestComponent {
+  constructor(element: ReactElement) {
+    this._currentElement = element;
+    this._renderedChildren = null;
+    this._topLevelWrapper = null;
+  }
 
-// =============================================================================
-
-var ReactTestComponent = function(element) {
-  this._currentElement = element;
-  this._renderedChildren = null;
-  this._topLevelWrapper = null;
-};
-
-ReactTestComponent.prototype.mountComponent = function(
-  transaction,
-  nativeParent,
-  nativeContainerInfo,
-  context
-) {
-  var element = this._currentElement;
-  this.mountChildren(element.props.children, transaction, context);
-};
-
-ReactTestComponent.prototype.receiveComponent = function(
-  nextElement,
-  transaction,
-  context
-) {
-  this._currentElement = nextElement;
-  this.updateChildren(nextElement.props.children, transaction, context);
-};
+  mountComponent(
+    transaction: ReactTestReconcileTransaction,
+    nativeParent: null | ReactTestComponent,
+    nativeContainerInfo: ?null,
```

Comment:
I wasn't sure how to type `nativeParent` and `nativeContainerInfo`. From what I could tell, `nativeParent` is either null or another instance of `ReactTestComponent`, and `nativeContainerInfo` was either `null` or `undefined`.